python/lsst/ts/salobj/sal_info.py

```diff
@@ -287,7 +310,7 @@ def add_reader(self, topic):
         Parameters
         ----------
         topic : `topics.ReadTopic`
-            Reader topic.
+            Read topic.
```

Comment:
The reason I have the Parameters section is to document the data type in a standard way.

Reply:
Is "The topic to be read from." a better docstring in these cases?

Reply:
I'll use:
```
        topic : `topics.ReadTopic`
            Topic to read and (eventually) close.
```
and for add_writer (since SalInfo does not write to it):
```
        topic : `topics.WriteTopic`
            Write topic to (eventually) close.
```